src/legacy/core_plugins/inspector_views/public/requests/details/req_details_stats.js

```diff
@@ -65,7 +65,10 @@ class RequestDetailsStats extends Component {
     const sortedStats = Object.keys(stats).sort().map(name => ({ name, ...stats[name] }));
     // TODO: Replace by property once available
     return (
-      <EuiTable style={{ tableLayout: 'auto' }}>
+      <EuiTable
+        style={{ tableLayout: 'auto' }}
```

Comment:
I'd recommend removing this inline style as well. I tested how it affects tables in our docs, and it looks like it will prevent any text-wrapping if contents gets very long.
<img src="https://d.pr/free/i/Cw2Nu5+" />